README.md

```diff
@@ -79,6 +79,28 @@ The server supports the following queries:
 
 Note that degrees of support for each LSP request are varying.
 
+## Contribution
+
+```bash
+# clone repo with submodules
+git clone --recursive git@github.com:ocaml/ocaml-lsp.git
+
+# if you already clone, pull submodules
+git submodule update --init --recursive
+
+# creat local switch (or use global one)
+opam sw create . 4.09.0
```

Comment:
It's probably more clear to newcomers to use switch instead of sw. (This is just a suggestion, feel free to ignore it.)